lms/static/lms/js/spec/main.js

```diff
@@ -51,7 +51,7 @@
             'squire': 'xmodule_js/common_static/js/vendor/Squire',
             'jasmine-imagediff': 'xmodule_js/common_static/js/vendor/jasmine-imagediff',
             'domReady': 'xmodule_js/common_static/js/vendor/domReady',
-            'mathjax': '//cdn.mathjax.org/mathjax/2.6-latest/MathJax.js?config=TeX-MML-AM_SVG&delayStartupUntil=configured', // jshint ignore:line
+            mathjax: '//cdn.mathjax.org/mathjax/2.6-latest/MathJax.js?config=TeX-MML-AM_SVG&delayStartupUntil=configured',  // eslint-disable-line max-len
```

Comment:
we decided to adopt this style in a fedx meeting a while back, we discussed it a bit here https://github.com/edx/eslint-config-edx/issues/10 and @dsjen merged it here: https://github.com/edx/eslint-config-edx/pull/13